editor/ScreenLayers/ProjectMenu.cs

```diff
@@ -79,7 +79,7 @@ public override void Load()
                 Fill = true,
                 Children = new Widget[]
                 {
-                    timeLabel = new Label(WidgetManager)
+                    timeLabel = new Button(WidgetManager)
                     {
                         StyleName = "light",
```

Comment:
There's no `light` style defined, see resources/skin.json, probably want to use the `small` one